packages/formula-editor/src/cursor.ts

```
@@ -0,0 +1,157 @@
+        }
+      }
+    }
+    return pos;
+  };
+  
+  static getCursorRect(shadowRoot: ShadowRoot) {
+    const range = Cursor._getComposedRange(shadowRoot);
+    return range ? range.getClientRects()[0] : undefined;
+  }
+  
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)